source-dropbox/oauth2.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/dropbox/dropbox-sdk-go-unofficial/v6/dropbox"
+	"golang.org/x/oauth2"
+)
+
+// TODO: Replace the client ID, client secret and redirect URL with the real ones
```

Comment:
These will need to come from the `connectors` table. We won't be able to put these values directly into the code. I'm a little hazy on how this is all threaded together, but it should work similar to `materialize-google-sheets`, or maybe even a better example is `source-google-drive`.

Reply:
Okay, I'm just trying to understand how do I access the OAuth2.0 Token from the code so I can use it in the client instantiation.